Vagrantfile

```diff
@@ -47,30 +36,31 @@ $script = <<EOF
    if [ ! -d /etc/puppet/modules/java ]; then
       puppet module install 7terminals-java
    fi
-
 EOF
 
-   config.vm.provision :shell do |shell|
-      shell.inline = $script
-   end
+      server_config.vm.provision :shell do |shell|
+         shell.inline = $script
+      end
 
-   config.vm.provision :puppet do |puppet|
-      puppet.manifests_path = 'puppet/manifests'
-      puppet.manifest_file = 'site.pp'
-      puppet.module_path = ['/home/akila/Documents/WSO2/WSO2-GitHub/Puppet-Modules.git',
+      server_config.vm.provision :puppet do |puppet|
+         puppet.manifests_path = 'puppet/manifests'
+         puppet.manifest_file = 'site.pp'
+         puppet.module_path = ['/home/rajkumar/projects/wso2/public/puppet-modules',
                             '/etc/puppet/modules']
```

Comment:
Can't we move this module path also to the configuration? Then we can use this with different environment without changing the vagrant file.